src/yz_extractor.erl

```diff
@@ -31,6 +31,7 @@
 -define(DEFAULT_MAP, [{default, yz_noop_extractor},
                       {"application/json",yz_json_extractor},
                       {"application/riak_counter", yz_dt_extractor},
+                      {"application/riak_gset", yz_dt_extractor},
                       {"application/riak_map", yz_dt_extractor},
```

Comment:
Wish this was more robust. the presumption that the list is sorted is a bit iffy.